dashboard/v1.1/src/pages/Dashboard/CPUUsage.tsx

```diff
@@ -58,6 +58,9 @@ const CPUUsage: FC<CPUUsageProps> = ({ cpuMetrics }) => {
     },
     theme: {
       mode: themeMode
+    },
+    xaxis: {
+      tickAmount: 12
```

Comment:
can we accept this from input as a dropdown in an interactive and compact way?

Reply:
Okay, Thankyou so much, I am working on it.